src/jsonapi-vuex.js

```diff
@@ -359,6 +359,37 @@ const jsonapiModule = (api, conf = {}) => {
 
 // Helper methods
 
+const addJvHelpers = (obj) => {
+  // Add Utility functions to _jv child object
+  Object.assign(obj[jvtag], {
+    is_rel(name) {
```

Comment:
Should we use camel case for these?

Reply:
haha - old habits die hard! Quite right!